tests/test_config.py

```diff
@@ -62,6 +62,42 @@ def test_valid_toml_two(
     assert not output.err
 
 
+def test_toml_style(
+    capsys: pytest.CaptureFixture[str], monkeypatch: pytest.MonkeyPatch
+) -> None:
+    """Test a correct toml with write = False."""
+    monkeypatch.chdir(CONFIG_DATA / "valid_toml_two")
+    pydocstringformatter.run_docstring_formatter(["test_package"])
+    # We are assuming the default style is "pep257"
+    default_output = capsys.readouterr()
+    assert default_output.out.endswith(
+        '''
+@@ -1,3 +1,2 @@
+-"""
+-A docstring"""
++"""A docstring."""
+ 
+'''
+    )
+
+    with open("pyproject.toml", "a", encoding="utf8") as file:
+        file.write("style = 'numpydoc'")
+
+    pydocstringformatter.run_docstring_formatter(["test_package"])
+    numpydoc_output = capsys.readouterr()
+
+    assert numpydoc_output.out.endswith(
```

Comment:
Thanks for spotting this issue!

For this docstring `numpydoc` and `pep257` shouldn't have different output. I have pushed https://github.com/DanielNoord/pydocstringformatter/pull/158 to address this.
Could you merge `main` into your branch and update this test? I think taking inspiration from `TestStyleOption` would be best. Those tests show how to check that the `config` attribute is set correctly.